app/shared/components/Tools/Form/CreateAccount.js

```diff
@@ -369,7 +377,7 @@ class ToolsFormCreateAccount extends Component<Props> {
                     <Message
                       content={t('tools_form_create_account_delegated_resources_warning')}
                       icon="info circle"
-                      warning
+                      info
```

Comment:
I changed this to info, because the message wasn't displaying at all with warning.

Reply:
That's because the form itself needs the `errors` flag on it, which is a boolean indicating whether to display warnings or not. You should be able to change this back to a warning, and use the same "has errors" logic to determine whether the `errors` flag on the form is true/false.

Reply:
I see, but this will get displayed even when there is no errors. I could make a new hasWarning variable though that I could use on the form.

Reply:
You can still use your same display logic on whether the element itself is rendered or not, but if you want a `warning` message to display within a form, the `error` value has to be set to true. 

Using `info` doesn't make sense because this is a warning, and it should look like a warning, not an info message.